### ISSUE TITLE
Bug: Chosen Checkpoint or Lora is displayed as [object Object]

### ISSUE THREAD

**Issue by bizzou** [pythongosssss/ComfyUI-Custom-Scripts#152]
Whenever I use your (brilliant) nodes to load checkpoints or Loras, the UI just gives me [object Object]. This is of course not very helpful. It may have something to do with me working on a Mac but I don't know. I assume that some browsers or python variants default to a different string representation of an object. Please make it explicit, what string is to be used, if at all possible.

Thanks for your great work.

**Comment by unphased** [comment]
yeah this is weird, seems like it works fine in chrome but i get this in safari too.

**Comment by miurgh** [comment]
I have a problem with Safari on my Mac.
